test/runtime/configMatters/comm/cache-remote/miniUnorderedCopyStress.chpl

```diff
@@ -0,0 +1,54 @@
+use BlockDist;
+
+// This is a simplified version of the unorderedCopyStress.chpl test
+// that is focused on testing a cache-remote issue with yielding
+// while in a cache operation (which can happen under ugni comms).
+
+config const oversubscription = 10,
+             tasksPerLocale = here.maxTaskPar * oversubscription;
+
+config const sizePerLocale = 10000,
+             size = sizePerLocale * numLocales;
+
+const space = {0..size};
+const D = space dmapped Block(space, dataParTasksPerLocale=tasksPerLocale);
+var A, reversedA: [D] int;
+
+forall (a, r, d) in zip(A, reversedA, D) {
+  a += d:int;
+  r += d:int;
```

Comment:
Doesn't really matter since it's just a test, but this and some of the other oddities were to support tuples. I think you could just do `var A, reversedA: [D] int = D;` instead. And start could just be `reversedA = 0` (and similar undoing of changes from https://github.com/chapel-lang/chapel/pull/14230/files#diff-291613c4514dbe80dfc597b1a880bc59)

Also see https://github.com/chapel-lang/chapel/pull/14252 for some context on the current problem size. If you wanted, I think you could bump the problem size to those old values. 

Up to you if you want to make either of these changes. So long as the test stresses the code that was failing before, it's good with me.

Reply:
I checked that the test failed before the changes in this PR and I don't want to do that again with these minor changes, so I'll leave it as is for now.